marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -296,3 +296,16 @@ module.exports.updateJob = module.amp(
   }
 );
 
+module.exports.deleteJobs = module.amp(
```

Comment:
Something to consider during beta testing is whether this should become a public API; it might be nice to expose this for Pari to make use of however she wants. I think we'd put it under data-hub/public/flow/jobApi.sjs, something like that.